internal/peer/chaincode/common_test.go

```diff
@@ -175,8 +175,7 @@ func TestGetOrdererEndpointFromConfigTx(t *testing.T) {
 	ordererEndpoints, err := common.GetOrdererEndpointOfChain(mockchain, signer, mockEndorserClient, cryptoProvider)
 	assert.NoError(t, err, "GetOrdererEndpointOfChain from genesis block")
 
-	assert.Equal(t, len(ordererEndpoints), 1)
-	assert.Equal(t, ordererEndpoints[0], "127.0.0.1:7050")
+	assert.Equal(t, len(ordererEndpoints), 0)
```

Comment:
This function is currently broken and will be addressed in [FAB-18055](https://jira.hyperledger.org/browse/FAB-18055)

Reply:
Would be nicer to actually have this in the commit rather than GH, but, not the end of the world.